app/assets/stylesheets/common/select-kit/toolbar-popup-menu-options.scss

```diff
@@ -2,7 +2,7 @@
   &.dropdown-select-box {
     &.toolbar-popup-menu-options {
       .select-kit-collection {
-        max-height: 50vh;
+        max-height: calc(max(var(--composer-height), 100vh) * 0.33);
```

Comment:
Hmm, not sure we need to use `max()` here, `max(var(--composer-height), 100vh)` will essentially default to `100vh` because the composer height will never exceed the viewport height, so the calculation will always give `33vh`.

Reply:
🤦🏽‍♂️ ah yes you're right. Should I just make it `max-height: 33vh` or do you think `max-height: 40vh` works better?

Reply:
I'd like to try with something like: 

```
max-height: calc(var(--composer-height) - 120px);
```

Note that `120px` is most certainly not the correct value, but basically, we want the height of the composer minus buttons/fields. Note also that these will be different for the reply composer (smaller) and topic creation (larger).

Reply:
As I'm testing this, I'm wondering if basing only off of `composer-height` and not the viewport will work correctly.

For example, increasing the developer tools size in height pushes the composer size smaller but the custom property is not updated. Still causing issues with the popup menu.

Reply:
Hmm... good point. I wonder whether we should also fix that (updating `composer-height` when it changes due to window resizing).